install.sh

```diff
@@ -154,7 +126,22 @@ case "${ARCH}" in
   ;;
 esac
 
-# create the device.json file if it doesn't exist
+# Get the URLs and hashes of the bootstrap packages
+for package in ruby git; do
+  [[ "$(curl "${URL}/packages/${package}.rb" | sed -n '/binary_sha256/,/}/p')" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
+    sha256s+=("${BASH_REMATCH[1]}")
+
+  name=$(curl "${URL}/packages/${package}.rb" | sed -n '3s/class\(.*\)<.*/\L\1/')
```

Comment:
Isn't `${package}` already contain the name of the package?

Reply:
Not sure what you mean.